_portfolios_projects/vermont-iee.md

```diff
@@ -17,7 +17,8 @@ project_url:
 learn_more:
 product_clients:
 resources:
-portfolio: Public Benefits
+portfolio: State & Local
+portfolio-permalink: /state-local/
```

Comment:
This was originally under `Public Benefits` but is `no-link`d there and not for State & Local so I thought putting it here would be better!